libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Analytics/SFSDKSalesforceAnalyticsManager.m

```diff
@@ -44,6 +44,7 @@
 static NSString * const kEventStoresDirectory = @"event_stores";
 static NSString * const kEventStoreEncryptionKeyLabel = @"com.salesforce.eventStore.encryptionKey";
 static NSString * const kAnalyticsOnOffKey = @"ailtn_enabled";
+static NSString * const kSFAppFeatureAiltnEnabled = @"AI";
```

Comment:
Just thinking about this over the course of these additions: it seems like these constants should be owned/aggregated in a common location somewhere.  I.e. it'd be nice to go to a single `SFSDKFeatureConstants.h` file and see the delineation of all possible feature registrations that the SDK exposes.  Thoughts?